pol-core/clib/cfgfile.cpp

```diff
@@ -986,27 +986,27 @@ void ConfigFile::display_error( const std::string& msg, bool show_curline,
                                 const ConfigElemBase* elem, bool error ) const
 {
   bool showed_elem_line = false;
-  fmt::Writer tmp;
-  tmp << ( error ? "Error" : "Warning" ) << " reading configuration file " << _filename << ":\n"
-      << "\t" << msg << "\n";
+  std::string tmp = fmt::format(
+      " {} reading configuration file {}:\n"
+      "\t{}",
+      error ? "Error" : "Warning", _filename, msg );
 
   if ( elem != nullptr )
   {
     if ( strlen( elem->type() ) > 0 )
     {
-      tmp << "\tElement: " << elem->type() << " " << elem->rest();
+      tmp += fmt::format( "\n\tElement: {} {}", elem->type(), elem->rest() );
       if ( _element_line_start )
-        tmp << ", found on line " << _element_line_start;
-      tmp << "\n";
+        tmp += fmt::format( ", found on line {}", _element_line_start );
       showed_elem_line = true;
     }
   }
 
   if ( show_curline )
-    tmp << "\tNear line: " << _cur_line << "\n";
+    tmp += fmt::format( "\n\tNear line: {}\n", _cur_line );
   if ( _element_line_start && !showed_elem_line )
-    tmp << "\tElement started on line: " << _element_line_start << "\n";
-  ERROR_PRINT << tmp.str();
+    tmp += fmt::format( "\n\tElement started on line: {}", _element_line_start );
```

Comment:
Could this cause two new-lines printed, if `show_curline` is true and the second conditional is false? There would be a trailing `\n` added to `tmp` and then an additional printed via `ERROR_PRINTLN`